server/server_test.go

```diff
@@ -1182,6 +1182,44 @@ func TestOIDCConfigChangeDetection_NoChange(t *testing.T) {
 	assert.Equal(t, result, false, "no error since no config change")
 }
 
+func TestOIDCConfigChangeDetection_WholeConfigFromSecret(t *testing.T) {
+	//Given
+	rawOIDCConfig := "$k8ssecret:oidcconfig"
+	originalConfig := settings_util.OIDCConfig{
+		ClientID:     "argocd",
+		ClientSecret: "sharedargooauthsecret"}
```

Comment:
maybe it would help to add a field that is not a clientID or clientSecret here? Just so it is easier to see how this block is testing something different from other unti tests that look like they could be doing the same thing

Reply:
Yeah good suggestion